examples/mipmap/main.rs

```diff
@@ -154,10 +154,7 @@ impl Example {
             mag_filter: wgpu::FilterMode::Linear,
             min_filter: wgpu::FilterMode::Nearest,
             mipmap_filter: wgpu::FilterMode::Nearest,
-            lod_min_clamp: 0.0,
-            lod_max_clamp: 100.0,
-            compare: None,
-            anisotropy_clamp: None,
+            .. wgpu::SamplerDescriptor::default()
```

Comment:
This might cause conflicts with #350 as that does this same thing, but less aggressively.